plugins/woocommerce/src/Internal/DataStores/Orders/DataSynchronizer.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * DataSynchronizer class file.
+ */
+
+namespace Automattic\WooCommerce\Internal\DataStores\Orders;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * This class handles the data migration/synchronization for the custom orders table. Its responsibilites are:
+ *
+ * - Performing the initial table creation and filling (triggered by initiate_regeneration)
+ * - Syncrhonizing changes between the custom orders table and the posts table whenever changes in orders happen.
```

Comment:
minor typo
```suggestion
 * - Synchronising changes between the custom orders table and the posts table whenever changes in orders happen.
```

Reply:
Interesting... looks like the American spelling uses the "z" and the international spelling uses the "s" 🤔

I haven't found other appearances of the word in Woo core, but Blocks, Select2 and Jetpack use the "z" variant so perhaps it's better to use it here as well for consistency?

Reply:
I mean the typo with the `nrch` -> `nchr`, but seems like it also "corrected" z to s. Updated the suggestion.